packages/jest-diff/src/diffStrings.js

```diff
@@ -34,20 +42,18 @@ function diffStrings(a: string, b: string): ?string {
 
       const lines = part.value.split('\n');
       const color = part.added
-        ? chalk.green
-        : (part.removed ? chalk.red : chalk.white);
+        ? chalk.red
+        : (part.removed ? chalk.green : chalk.white);
 
       if (lines[lines.length - 1] === '') {
         lines.pop();
       }
 
       return lines.map(line => {
-        const mark = part.added
-          ? chalk.green('+')
-          : part.removed ? chalk.red('-') : ' ';
+        const mark = color(part.added ? '+' : part.removed ? '-' : ' ');
```

Comment:
green `-`s and red `+`s might still be confusing i feel. But let's see how it works